infrastructure/kube/keep-test/electrs-configmap.yaml

```diff
@@ -0,0 +1,8 @@
+kind: ConfigMap
```

Comment:
Same comment as in: https://github.com/keep-network/electrs/pull/1#discussion_r704203843